followthemoney/graph.py

```diff
@@ -39,7 +49,9 @@ def to_dict(self):
         }
 
     @classmethod
-    def from_proxy(cls, proxy):
+    def from_proxy(cls, proxy: EntityProxy) -> 'Node':
+        if proxy.id is None:
+            raise Exception('<FIXME>')
```

Comment:
FIXME :)

Reply:
Yeah, I wasn't sure which kind of exception I should be using ;)